### PR TITLE
PRODSEC-1755 Whitesource Transition to Snyk

### PR DESCRIPTION
**Pull request by sfc-gh-jfan**

### CODE REVIEW

File: .github/workflows/snyk-pr.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        fetch-depth: 0
+
+    - name: checkout action
+      uses: actions/checkout@v3
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
```
Comment (sfc-gh-hchaturvedi) [inline .github/workflows/snyk-pr.yml:21]: Again why are we using WHITESOURCE this often. This can cause confusion.

Comment (sfc-gh-jfan) [reply .github/workflows/snyk-pr.yml:21]: Hey, @sfc-gh-hchaturvedi  the WHITESOURCE_ACTION_TOKEN means the github token to the script we use for automation. The repo name is called `whitesource-action` => https://github.com/snowflakedb/whitesource-actions

it actually has nothing to do with real whitesource app

File: .github/workflows/snyk-issue.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout action
+      uses: actions/checkout@v3
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
```
[the remark this answers is not in the document]
Comment (sfc-gh-jfan) [reply .github/workflows/snyk-issue.yml:17]: added comment above

**Comment by sfc-gh-jfan** [comment]
Hey, @sfc-gh-hchaturvedi  I have addressed your comments to add more descriptive GHA job names to the yaml file, could you pls review it? thanks!

**Comment by sfc-gh-hchaturvedi** [comment]
Just so it gets documented, though there are instances of the term WHITESOURCE sprinkled around, the change has nothing to do with WHITESOURCE. The repository unfortunately is under the name and it is apparently cumbersome to update it. This merge moves the security tool from Whitesource to Snyk

**Review by sfc-gh-hchaturvedi — APPROVED** [review]
LGTM. But I hope we can update the name of the sec scan repo from Whitesource to something more generic.

**Pull request merged**